### PR TITLE
Include the clientID in the store method

### PR DESCRIPTION
**Pull request by yangli2014**
Add the clientID in the store method signature

### CODE REVIEW

File: plugin/src/main/java/org/opennms/timeseries/cortex/CortexTSS.java [hunk 1]
```diff
@@ -96,7 +96,7 @@
  *
  * @author jwhite
  */
-public class CortexTSS implements TimeSeriesStorage {
```
Comment (j-white) [inline plugin/src/main/java/org/opennms/timeseries/cortex/CortexTSS.java:99]: The class must continue to implement the `TimeSeriesStorage` interface.

File: plugin/src/main/java/org/opennms/timeseries/cortex/CortexTSS.java [hunk 2]
```diff
@@ -165,8 +165,7 @@ public CortexTSS(final CortexTSSConfig config) {
         metrics.register("maxAllowedConcurrentCalls", (Gauge<Integer>) () -> asyncHttpCallsBulkhead.getMetrics().getMaxAllowedConcurrentCalls());
     }
 
-    @Override
-    public void store(final List<Sample> samples) throws StorageException {
+    public void store(final List<Sample> samples, String clientID) throws StorageException {
```
Comment (j-white) [inline plugin/src/main/java/org/opennms/timeseries/cortex/CortexTSS.java:168]: Let's add a new method instead, and have the existing method call in the the value from the config.

We can take a similar approach with the other methods such as `findMetrics` and `getTimeseries`.

Comment (yangli2014) [reply plugin/src/main/java/org/opennms/timeseries/cortex/CortexTSS.java:168]: I see, good idea. How those findMetrics and getTimeseries methods using the client ID? I didn't find information in the cortex API document.

Comment (j-white) [reply plugin/src/main/java/org/opennms/timeseries/cortex/CortexTSS.java:168]: We need to set the value of the `X-Scope-OrgID` header for all the HTTP requests we make.

File: plugin/src/main/java/org/opennms/timeseries/cortex/CortexTSS.java [hunk 3]
```diff
@@ -167,6 +167,9 @@ public CortexTSS(final CortexTSSConfig config) {
 
     @Override
     public void store(final List<Sample> samples) throws StorageException {
+        store(samples, config.getOrganizationId());
+    }
+    public void store(final List<Sample> samples, String clientID) throws StorageException {
```
Comment (j-white) [inline plugin/src/main/java/org/opennms/timeseries/cortex/CortexTSS.java:172]: Nit: formatting, add new line between function end and the start of the next

**Review by j-white — APPROVED** [review]
Looks good - this should do the trick!

**Pull request merged**